test.js

```diff
@@ -1,4 +1,5 @@
 require('./ofe').call();
 
-var str = '1';
-while (true) str += str;
+var i = 0;
+var a = {};
+while (true) a[++ i] = i;
```

Comment:
In versions node@0.12 and iojs was getting this error:
```
$ node test.js 
/home/sgimeno/node/node-ofe/test.js:4
while (true) str += str;
                  ^
RangeError: Invalid string length
    at Object.<anonymous> (/home/sgimeno/node/node-ofe/test.js:4:19)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```
Instead of a Fatal Error